deb/build/debian/control

```diff
@@ -1,12 +1,12 @@
 Source: @@ARTIFACTNAME@@
 Section: devel
-Priority: extra
+Priority: optional
 Maintainer: @@AUTHOR@@
-Build-Depends: debhelper (>= 9)
-Standards-Version: 3.9.4
+Build-Depends: debhelper (>= 10)
+Standards-Version: 3.9.8
```

Comment:
Moving from [August 2012 to April 2016](https://www.debian.org/doc/packaging-manuals/upgrading-checklist.txt).